docs/rules/parameter-properties.md

```diff
@@ -4,8 +4,18 @@
 
 Enforces that code doesn't use TypeScript's class parameter properties:
 
+# Invalid Code
+
 ```ts
 class Values {
 	constructor(readonly value: number) {}
 }
 ```
+
+# Valid Code
+
+```ts
+class Values {
+	constructor(value: number) {}
```

Comment:
[Docs] _Valid_ examples generally should be as close in functionality to the _Invalid_ ones as possible. Otherwise it's confusing for less-confident readers:

```suggestion
  readonly value: number;
	constructor(value: number) {}
```

Reply:
Thanks. I'll fix this.

My initial idea is to use the parent prop and insertBefore(...) to attach the property